dist/fyta-plant-card.js

```diff
@@ -1070,25 +1024,27 @@ class FytaPlantCard extends LitElement {
   }
 
   _buildNutritionTooltipContent(statusState, daysUntilFertilization, lastFertilizationDateString, nextFertilizationDateString) {
-    let tooltipContent = `Nutrition Status: ${statusState.replace(/_/g, ' ')}`;
+    const tooltipContentLine1 = `Nutrition Status: ${statusState.replace(/_/g, ' ')}`;
+    let tooltipContentLine2 = '';
+    let tooltipContentLine3 = '';
 
     if (daysUntilFertilization !== null && !isNaN(daysUntilFertilization)) {
       const daysText = Math.abs(daysUntilFertilization) === 1 ? 'day' : 'days';
       if (daysUntilFertilization >= 0) {
-        tooltipContent += `<br>Fertilize in ${daysUntilFertilization} ${daysText}`;
+        tooltipContentLine2 = `Fertilize in ${daysUntilFertilization} ${daysText}`;
       } else {
-        tooltipContent += `<br>Fertilization overdue by ${Math.abs(daysUntilFertilization)} ${daysText}`;
+        tooltipContentLine2 = `Fertilization overdue by ${Math.abs(daysUntilFertilization)} ${daysText}`;
       }
 
       if (lastFertilizationDateString) {
-        tooltipContent += `<br>Last Fertilization: ${this._formatDateForDisplay(lastFertilizationDateString)}`;
+        tooltipContentLine3 = `Last Fertilization: ${this._formatDateForDisplay(lastFertilizationDateString)}`;
       }
       if (nextFertilizationDateString) {
-        tooltipContent += `<br>Next Fertilization: ${this._formatDateForDisplay(nextFertilizationDateString)}`;
+        tooltipContentLine3 = `Next Fertilization: ${this._formatDateForDisplay(nextFertilizationDateString)}`;
       }
```

Comment:
```suggestion
        if (nextFertilizationDateString) {
          tooltipContentLine3 = `Last Fertilization: ${this._formatDateForDisplay(lastFertilizationDateString)}`;
        } else if (lastFertilizationDateString) {
          tooltipContentLine3 = `Last Fertilization: ${this._formatDateForDisplay(lastFertilizationDateString)}`;
        }
```
Would it make sense to put this in an `else if`-clause, as only either of both is displayed and nextFertilization prevails?

Reply:
Good point. Will adjust accordingly.